lightning: disk quota GA (#11409)
pingcap/docs: tidb-lightning/tidb-lightning-configuration.md

# node in the physical import mode. 0 by default, which means no limit.
# store-write-bwlimit = "128MiB"

> # Specifies the disk quota for the local temporary files when using the Physical Import Mode.
> # When the disk quota is insufficient, TiDB Lightning pauses the process of reading the source data and writing the temporary files,

Reviewer: ```suggestion
# Specifies the disk quota for local temporary files when physical import mode is used.
# When the disk quota is insufficient, TiDB Lightning stops reading source data and writing temporary files,
```

> # and prioritizes writing the already sorted key-value pairs to TiKV.

Reviewer: ```suggestion
# but prioritizes writing the already sorted key-value pairs to TiKV.
```

switch-mode = "5m"
# Duration between which an import progress is printed to the log.
log-progress = "5m"

> # When you use the Physical Import Mode, the time interval for checking the local disk quota.

Reviewer: ```suggestion
# The time interval for checking the local disk quota when you use the physical import mode.
```
